layout/ut/plan.c

```diff
@@ -0,0 +1,309 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2012-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
This is a new file. So,  the start year should be 2021, I think.

Reply:
Fixed.